### PR TITLE
docs: minor update to "set up fcc locally guidelines" - adding 'closing quotes' to docker set up

### PR DESCRIPTION
**Pull request by Hash2C**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

On "how-to-setup-freecodecamp-locally.md".
Minor fix, helps new contributors to copy/paste the commands.
Just added some closing `"` that were missing.
Depending on the Docker configuration, it might be needed to prepend the commands with `sudo`, I wonder if some "tip" should be included for this case.

Going for the first freeCodeCamp pull request. Also looking for feedback. woohoo!

### CODE REVIEW

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -291,10 +291,10 @@ A quick reference to the commands that you will need when working locally.
...
 | `npm run docker:develop` | Start the freeCodeCamp API Server and Client Applications. |
 | `npm run docker:test:init` | Bootstrap the test container, necessary for testing in docker. |
-| `npm run docker:test -- -c "npm run test` | Run all JS tests in the system, including client, server, lint and challenge tests. |
-| `npm run docker:test -- -c "npm run test:curriculum` | Run the curriculum test suite. |
-| `npm run docker:test -- -c "npm run test:client` | Run the client test suite. |
-| `npm run docker:test -- -c "npm run test:server` | Run the server test suite. |
+| `npm run docker:test -- -c "npm run test"` | Run all JS tests in the system, including client, server, lint and challenge tests. |
+| `npm run docker:test -- -c "npm run test:curriculum"` | Run the curriculum test suite. |
+| `npm run docker:test -- -c "npm run test:client"` | Run the client test suite. |
+| `npm run docker:test -- -c "npm run test:server"` | Run the server test suite. |
```
Comment (Hash2C) [inline docs/how-to-setup-freecodecamp-locally.md]: Gourgeous stuff xD

Comment (sithsupersu) [reply docs/how-to-setup-freecodecamp-locally.md]: > Just exploring. Gourgeous stuff xD

Thank you for your cleanup

Comment (sithsupersu) [reply docs/how-to-setup-freecodecamp-locally.md]: thanks will test

Comment (Hash2C) [reply docs/how-to-setup-freecodecamp-locally.md]: it surely is a pleasure to contribute

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by ojeytonwilliams** [comment]
@Hash2C Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Comment by Hash2C** [comment]
:tada::tada::tada:
thanks @ojeytonwilliams

**Pull request merged**